src/lib/save_result.ml

```diff
@@ -63,7 +63,7 @@ let make_saving_node
       )
   in
   let name =
-    sprintf "Salvation: %s" (Filename.basename path) in
+    sprintf "Saved: %s" key in
```

Comment:
@ihodes @arahuja please agree on a name :)
I'm doing another PR (to save Seq2HLA)